ANDROID: epoll_wait06: Use kernel_page_size for EPOLLET

The semantics of this test don't hold across page boundaries for EPOLLET.
The documentation says that it "might" / "probably" won't receive a
new event for existing data, so we can't be guaranteed that it won't.
Use kernel_page_size() instead of getpagesize() to handle 16KB page size
emulation correctly.

Test: atest vts_ltp_test_x86_64:syscalls.epoll_wait06_64bit

diff --git a/testcases/kernel/syscalls/epoll_wait/epoll_wait06.c b/testcases/kernel/syscalls/epoll_wait/epoll_wait06.c
index 734d803d5..6e259fb15 100644
--- a/testcases/kernel/syscalls/epoll_wait/epoll_wait06.c
+++ b/testcases/kernel/syscalls/epoll_wait/epoll_wait06.c
@@ -24,6 +24,7 @@
 #include <fcntl.h>
 #include "tst_test.h"
 #include "tst_epoll.h"
+#include "pgsize_helpers.h"
 
 static size_t write_size;
 static size_t read_size;
@@ -32,7 +33,16 @@
 
 static void setup(void)
 {
-	write_size = getpagesize();
+	/*
+	 * NOTE: The sematics of this test don't hold across page boundaries for
+	 *       EPOLLET.
+	 *
+	 * The documentation says that it "might" / "probably" won't recieve a
+	 * new event for exisitng data. So we can't be guaranteed that it won't.
+	 *
+	 * https://man7.org/linux/man-pages/man7/epoll.7.html
+	 */
+	write_size = kernel_page_size();
 	read_size = write_size / 2;
 
 	SAFE_PIPE2(fds, O_NONBLOCK);
